app/controllers/assignments_controller.rb

```diff
@@ -77,7 +77,7 @@ def destroy
   end
 
   def assistant
-    code_param = CGI.escape(current_user.api_token)
+    code_param = current_user.api_token
```

Comment:
Moved URL encoding to base `MessageVerifier` class so that we don't have to do this everytime we generate a token